Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.h

```diff
@@ -73,6 +73,7 @@ enum class SDKAlignedBehavior {
     ObservesClassProperty,
     PictureInPictureMediaPlayback,
     ProcessSwapOnCrossSiteNavigation,
+    RemoveImageSelectionInEditablePage,
```

Comment:
Nit: I think we should specify that this is only on right click, and use `Prevent` instead of `Remove`.

Maybe a little verbose, but I propose `PreventImageSelectionOnContextualMenuClickInEditablePage`.

Reply:
Thank you for the suggestion! I was struggling to name this